### DISCUSSION TITLE
What's the right way to auto-convert pasted clipboard markdown content to yoopta blocks?

### DISCUSSION THREAD

**Discussion by trancethehuman** [yoopta-editor/Yoopta-Editor#302] [Q&A] [answered]
By default HTML converts automatically, but markdown doesn't. Wondering if there's an idiomatic way to do so?

**Comment by Darginec05** [comment]
@trancethehuman I was thinking about it and I got an idea.
I can't add markdown deserialize directly to "@yoopta/editor" because it will increase the package size. This will happen because markdown uses "marked" package which cost **+11 kB** 

But what if I add new prop (something like `parsers`) to the `<YooptaEditor/>`, which means that we add rules for analyzing pasted content from the clipboard.

Example: 
```tsx
import YooptaEditor, { createYooptaEditor } from '@yoopta/editor';
import { markdown, html, email } from '@yoopta/exports'
import { useMemo } from 'react';

const BasicExample = () => {
  const editor: YooEditor = useMemo(() => createYooptaEditor(), []);

  return (
        <YooptaEditor
         ...other props
          parsers={{
              html: html, 
              markdown: markdown, 
              email: email
          }}
        />
  );
};

```

And the editor will analyze the format in which the contents from the clipboard are inserted, and deserialize it using the parser prop into the Yoopta content

**Comment by trancethehuman** [comment]
Yeah that sounds great!

**Reply by Darginec05** [reply, marked as answer]
Cool! I'll convert this into issue
